scripts/lints

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# shellcheck disable=SC2016
+find . -type f -name go.mod | sort | while read -r mod; do
+  dir="$(dirname "$mod")"
+  name="$(basename "$dir")"
+  # get the parent directory when the module is nested semver-style
+  if [[ "$name" =~ ^v[0-9]+.*$ ]]; then
+    ver="$(basename "$dir")"
+    dir="$(dirname "$dir")"
+    name="$(basename "$dir")-$ver"
+  fi
+
+  moddir="$(dirname "$mod")"
+
+  echo "# auto-generated by scripts/builds. DO NOT EDIT.
```

Comment:
```suggestion
  echo "# auto-generated by scripts/lints. DO NOT EDIT.
```

could also do both lint and build workflows in the ./scripts/builds file 🤔

Reply:
Yeah, less code to maintain